google-cloud-tools-plugin/ultimate/src/com/google/cloud/tools/intellij/appengine/server/instance/AppEngineServerModel.java

```diff
@@ -162,14 +162,20 @@ public void checkConfiguration() throws RuntimeConfigurationException {
           "App Engine facet not found in '" + artifact.getName() + "' artifact");
     }
 
+    if (CloudSdkService.getInstance().getSdkHomePath() == null) {
+      throw new RuntimeConfigurationError(
```

Comment:
Why a `RuntimeConfigurationError` and not a `RuntimeConfigurationWarning`, as above?